x-pack/plugins/apm/public/plugin.ts

```diff
@@ -162,7 +162,24 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
       order: 8500,
       euiIconType: 'logoObservability',
       category: DEFAULT_APP_CATEGORIES.observability,
-
+      meta: {
+        keywords: [
+          'RUM',
+          'Real User Monitoring',
+          'DEM',
+          'Digital Experience Monitoring',
+          'EUM',
+          'End User Monitoring',
+          'UX',
+          'Javascript',
+          'APM',
+          'Mobile',
+          'digital',
```

Comment:
"Digital" seems a little too general to me. Is there some UX-specific context to the term that I am missing?

cc @drewpost

Reply:
Generality was certainly what I was going for with this. With digital tied to so many things: digital performance, digital experience, digital transformation, I wanted to try and be general at the top of the funnel, so to speak, for discoverability reasons. cc @justinkambic

Reply:
👍